packages/playwright-core/src/client/page.ts

```diff
@@ -691,6 +691,9 @@ export class Page extends ChannelOwner<channels.PageChannel> implements api.Page
   async pause() {
     if (require('inspector').url())
       return;
+    this._browserContext.setDefaultNavigationTimeout(0);
+    this._browserContext.setDefaultTimeout(0);
```

Comment:
I wonder if we should restore the timeout back, now that this code triggers with just a library?

Reply:
We could, we don't have means to doing that though, also does not seem to be important?